src/runtime/virtcontainers/clh.go

```diff
@@ -455,6 +457,48 @@ func (clh *cloudHypervisor) hotPlugVFIODevice(device config.VFIODev) error {
 	return err
 }
 
+func (clh *cloudHypervisor) hotplugNetDevice(endpoint Endpoint) error {
+	clh.Logger().WithField("endpoint-type", endpoint).Debugf("Hotplug Endpoint of type %v", endpoint)
+
+	var tap, mac string
+
+	switch endpoint.Type() {
+	case VethEndpointType:
+		mac = endpoint.HardwareAddr()
+		if mac == "" {
+			return errors.New("Endpoint MAC address is empty")
+		}
+
+		netPair := endpoint.NetworkPair()
+		if netPair == nil {
+			return errors.New("net Pair to be added is nil, needed to get TAP path")
+		}
+
+		tap = netPair.TapInterface.TAPIface.Name
+		if tap == "" {
+			return errors.New("TAP path in network pair is empty")
+		}
+	default:
+		return fmt.Errorf("this endpoint(%v) is not supported", endpoint.Type())
+	}
+
+	cl := clh.client()
+	ctx, cancel := context.WithTimeout(context.Background(), clhHotPlugAPITimeout*time.Second)
+	defer cancel()
+
+	_, _, err := cl.VmmPingGet(ctx)
```

Comment:
Please remove this as this is not needed and might slow down the code in production.

Reply:
Why do we need to ping there? @BetaXOi

Reply:
This is a legacy code path from `clh.go` in general. I will have a PR to remove all  the `VmmPingGet` in Kata 1.x (kata-runtime repo).  We will port that change to kata 2.0 while we enable clh CI here.

@BetaXOi In this PR, you can safely assume that a call to `VmmPingGet` is not required. Thanks.

Reply:
`VmmPingGet` has been removed. thanks.